src/dom_util.js

```diff
@@ -68,7 +68,7 @@ const getRoot = function(node) {
  *     corresponding to node, if present.
  */
 const getActiveElement = function(node) {
-  const root = getRoot(node);
+  const root = getRootNode.call(node);
```

Comment:
Looking at this [jsperf](https://jsperf.com/function-versus-function-call-versus-function-apply/2), it looks like they should be ~ the same. I think it may have been slower in the past, but I'm guessing it can be just inlined into something smarter these days.